fem/gridfunc.cpp

```diff
@@ -4545,6 +4545,14 @@ real_t ExtrudeCoefficient::Eval(ElementTransformation &T,
    return sol_in.Eval(*T_in, ip);
 }
 
+void VectorExtrudeCoefficient::Eval(Vector &v, ElementTransformation &T,
+                                    const IntegrationPoint &ip)
+{
+   ElementTransformation *T_in =
+      mesh_in->GetElementTransformation(T.ElementNo / n);
```

Comment:
I'm concerned about division by zero here. I suggest placing an MFEM_VERIFY line in the VectorExtrudeCoefficient constructor to ensure n>0.

Reply:
Done in https://github.com/mfem/mfem/pull/4781/commits/38df8156b9dc20db8b06bbb0a75efd112511a5f7 👍